model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -207,18 +207,13 @@ message BundleApplication {
 // Either an absolute timestamp or a relative timestamp can represent a
 // scheduled execution time.
 message DelayedBundleApplication {
-  // Recommended time at which the application should be scheduled to execute
-  // by the runner. Times in the past may be scheduled to execute immediately.
-  // TODO(BEAM-8536): Migrate usage of absolute time to requested_time_delay.
-  google.protobuf.Timestamp requested_execution_time = 1;
```

Comment:
Are there other places that need to be changed (e.g. does this need to be a two-phased commit)?

Reply:
I have changed all the usage in this PR. Just talked to Rebo and Daniel, one left here is to generate go proto within this PR. I'll let you know once I finish.

Reply:
I have rebuilt the go proto. Waiting for tests pass.